UPGRADE-7.0.md

```diff
@@ -303,6 +303,7 @@ Security
  * Add argument `$badgeFqcn` to `Passport::addBadge()`
  * Add argument `$lifetime` to `LoginLinkHandlerInterface::createLoginLink()`
  * Require explicit argument when calling `TokenStorage::setToken()`
+ * Calling the constructor of `DefaultLoginRateLimiter` with an empty secret now throws an `InvalidArgumentException`
```

Comment:
Not sure about the wording here.

Maybe `Require a non-empty string for the secret when calling DefaultLoginRateLimiter::__constructor()` ?